pkgs/applications/misc/ptask/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, pkgconfig, makeWrapper, gtk3, json_c, taskwarrior }:
```

Comment:
You have taskwarrior here in the input of the function, but not in any buildinputs.